lib/bloc/world_clock_bloc.dart

```diff
@@ -1,32 +1,42 @@
 import 'package:flutter_bloc/flutter_bloc.dart';
 import 'package:word_clock/bloc/world_clock_event.dart';
 import 'package:word_clock/bloc/world_clock_state.dart';
+import 'package:word_clock/world_clock_repository/hive_repository.dart';
 
-class WorldClockBloc extends Bloc<WorldClockEvent, WorldClockState>{
-  List<DateTime> _timeList=[];
-  WorldClockBloc():super(LoadingState()){
+import '../models/world_clock_model.dart';
+
+class WorldClockBloc extends Bloc<WorldClockEvent, WorldClockState> {
+  List<WorldClockModel> _timeList = [];
```

Comment:
it's better to keep that data in separate `Repository` class (not in hive utility repo)
but as this particular data is just as simple array - you can keep it here